client/src/main/views/pointingRoom/components/mainContent/VotingArea.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PointCard from './PointCard';
+
+const VotingArea = () => {
+	const voteNumbers = [0, 0.5, 1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
Move this to a generate function inside of client/src/main/utils/mock-data/voteNumbers
(mock-data is a new folder, voteNumbers.js is the file name)

Reply:
make a prop for this component 'voteNumbers', The parent should pass the result of the new function I said above into the new prop and use that as the list you map out.